app/src/main/java/com/babylon/wallet/android/data/repository/stream/StreamRepository.kt

```diff
@@ -113,7 +112,7 @@ class StreamRepositoryImpl @Inject constructor(
             manifestClassFilter = filters.transactionClass?.toManifestClass()?.let {
                 StreamTransactionsRequestAllOfManifestClassFilter(
                     propertyClass = it,
-                    matchOnlyMostSpecific = filters.transactionClass != TransactionClass.Transfer
+                    matchOnlyMostSpecific = true
```

Comment:
I haven't follow the whole discussion in slack but do you need to add here a comment why we go with true?
Especially if this has to change in future

Reply:
explanation of this parameter can be found in api docs.

Reply:
if you mean [here](https://docs.radixdlt.com/docs/search?query=StreamTransactionsRequestAllOfManifestClassFilter&searchType=) or [here](https://radix-babylon-gateway-api.redoc.ly/) I can't find it.